src/ios/CDVSound.m

```diff
@@ -686,6 +714,7 @@ @implementation CDVAudioFile
 @synthesize resourcePath;
 @synthesize resourceURL;
 @synthesize player, volume;
+@synthesize player, rate;
```

Comment:
I'm getting a problem with line 717 due to "player" already being implemented in line 716.